step-api/src/main/java/org/jenkinsci/plugins/workflow/steps/StepDescriptor.java

```diff
@@ -110,7 +111,7 @@ public Step newInstance(Map<String,Object> arguments) throws Exception {
      * @throws UnsupportedOperationException if this descriptor lacks the ability to do such a calculation
      */
     public Map<String,Object> defineArguments(Step step) throws UnsupportedOperationException {
-        return DescribableHelper.uninstantiate(step);
+        return DescribableModel.uninstantiate_(step);
```

Comment:
(Seems your initial code drop in `symbol-plugin` did not go through the PR process so there is no easy way to comment on that code directly. Generally I would suggest starting with an empty repository and then filing PR 1 to add initial proposed content.)